### ISSUE TITLE
[Settings]: Screen Reader is not announcing selected color name with System colors buttons.

### ISSUE THREAD

**Issue by ghost** [microsoft/terminal#11255]
### Windows Terminal version (or Windows build number)

1.10.2383.0

### Other Software

**Test Environment:**
OS: Windows 11 Version 21H2 (OS Build 22000.168)
Screen Reader: Narrator
Tool: Accessibility Insight for Windows

### Steps to reproduce

**Repro Steps:**
1. Open Windows Terminal.
2. Open Settings page using "Ctrl + ',' ".
3. Open Narrator using 'Win + Ctrl + Enter' key.
4. Navigate to 'Color Schemes' tab and activate it.
5. Then navigate to all buttons present below System Color text and observe the issue.

**User Experience:**
Non-visual users are impacted here as they will not be able to know which color is selected as System colors. 

**Guideline Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships

**Attachments:**
[Screen Reader is not announcing selected color name with System colors buttons..zip](https://github.com/microsoft/terminal/files/7183186/Screen.Reader.is.not.announcing.selected.color.name.with.System.colors.buttons.zip)

### Expected Behavior

Screen Reader should announce selected color name with System colors buttons. i.e. if light gray is selected as foreground color Screen Reader should announce 'System Colors Group Foreground Light Gray button'.

### Actual Behavior

Screen Reader is not announcing selected color name with System colors buttons. i.e. if light gray is selected as foreground color Screen Reader is announcing 'Foreground button'.

Observation: Issue repro with NVDA Screen Reader.

**Comment by carlos-zamora** [comment]
This was fixed by https://github.com/microsoft/terminal/pull/16544 and is available on Windows Terminal version 1.20.10293 or greater
